js/wide-tables.jquery.js

```diff
@@ -132,6 +132,20 @@ $.fn.wideTables = function wideTablesPlugin(options) {
     $container.data('id7.wide-tables', wideTables);
   }
 
+  if (!(navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1)) {
+    if (navigator.platform === 'MacIntel') {
+      // on a mac we want to show the scroll bar as we cannot detect system preferences setting for showing/hiding scroll bar
+      // when mac users have "AUTOMATICALLY show scroll bars" set, the grid would not be able to horizontally scroll at all in non-safari browser(even with macgic trackpad).
```

Comment:
>macgig

I'm not sure if this is an awesome pun, or just a typo

Reply:
that's a typo :)

Reply:
it has `mac` in it, though!

Reply:
it's mac after all